java/java.sourceui/src/org/netbeans/modules/java/source/ui/OpenProjectFastIndex.java

```diff
@@ -550,7 +550,7 @@ private PI getCacheInfo(Project p) {
         private final int         size;
         private final Project     project;
         
-        NameIndex(Project p, FileObject root, String files, ArrayList indices) {
+        NameIndex(Project p, FileObject root, String files, List indices) {
```

Comment:
```suggestion
        NameIndex(Project p, FileObject root, String files, List<Object[]> indices) {
```

Reply:
Hey @mklaehn , thanks for the sharp eyes!

It's been changed and integrated..